master_controller:
    ros__parameters:
        verbose: true                                              # printing info about controller states if true
        goal_pose_pub_topic: /master/target                        # topic with goal_pose for nav2
        goal_pose_sub_topic: /goal_pose
        nav2_cmd_vel_topic: /master/warper_cmd_vel                 # output topic for drive_controller
        cmd_vel_topic: /commands/velocity                             # output topic for drive_controller
        use_emergency_stop: true                                   # robot stops near dynamic objects if true
        backward_linear_vel: -0.20                                 # linear x velocity for moving backward in recovery
        allow_driving_topic: /master/allow_driving
        feedback_topic: /navigate_to_pose/_action/feedback
        result_topic: /master_controller/result
        odom_topic: /master/odom
        aruco_topic: /aruco/found
        delta: 0.30

dispatcher:
    ros__parameters:
        verbose: true                                              # printing info about controller states if true
        goal_pose_topic: /goal_pose                                # topic with goal_pose for controller
        master_allow_driving_topic: /master/allow_driving
        slave_allow_driving_topic: /slave/allow_driving
        result_topic: /master_controller/result
               #         x     y      th   id   parent_id
        points: [+0.0, +0.0, 0.00,  0.,  0.,  # 0
                 +1.2, +0.0, 0.00,  1.,  0.,  # 1
                 +2.1, +0.0, 0.00,  2.,  1.,  # 2
                 +1.8, -2.2, 4.71,  3.,  2.,  # 3
                 -0.3, -1.9, 3.14,  4.,  3.,  # 4  
                 +0.0, -1.3, 1.57,  5.,  4.,  # 5
                 +0.0, -2.2, 4.71,  6.,  3.,  # 6
                 +2.1, -1.9, 0.00,  7.,  2.,  # 7
                 +1.8, +0.3, 1.57,  8.,  1.,  # 8
                 +1.2, +0.0, 3.14,  9.,  0.,  # 9
                 +0.9, +2.1, 3.14,  10., 1.,  # 10
                 -1.5, +2.5, 1.57,  11., 10., # 11
                 -1.5, +0.5, 4.71,  12., 10., # 12
                 -2.5, +0.5, 3.14,  13., 12., # 13
                 ]

        